### ISSUE TITLE
Question about Data set

### ISSUE THREAD

**Issue by IIM-D** [HKUDS/RLMRec#18]
Hello！ 👋
First question: Are the training set, test set, and validation set merely adjacency matrices of interactions? In other words, when replacing the dataset, is it sufficient to only create adjacency matrices composed of 0/1?

Second question: I noticed that the number of users and items in the original dataset is relatively similar. If the number of users is significantly smaller than the number of items, would this affect the model from the perspectives of collaborative filtering and mutual information maximization?

**Comment by Re-bin** [comment]
Hi 👋!

Thanks for your interest in RLMRec! 

Regarding your first question: yes, the difference between the training, validation, and test sets is that they consist of different (non-overlapping) subsets of user-item interactions from the same user-item graph. We use a 0-1 sparse matrix to store the dataset, which is a sufficient representation for different datasets in the field of collaborative filtering.

For your second question: I believe that mutual information maximization will also work in this situation because it is theoretically not affected by the number of nodes.

I hope the above answers are helpful to you :)

Best regards,
Xubin

**Comment by IIM-D** [comment]
Thanks! Have a nice life and good luck with your studies!
